modules/ROOT/pages/special-parameters.adoc

```diff
@@ -83,10 +83,12 @@ IMPORTANT: You need to consider the generic type. A parameter derived from uses
 [[parameter-resolver]]
 == ParameterResolver<Type>
 
-`ParameterResolver<Type>` allows you to defer the resolution of an expression. This is useful when a parameter that takes an expression as a value is not required. You can improve performance by resolving such an expression on demand from within the connectors's code instead of resolving it automatically.
+`ParameterResolver<Type>` allows you to defer the resolution of an expression on a Operation. This is useful when a parameter that takes an expression as a value is not required. You can improve performance by resolving such an expression on demand from within the connectors's code instead of resolving it automatically.
```

Comment:
```suggestion
`ParameterResolver<Type>` allows you to defer the resolution of an expression on an operation. This is useful when a parameter that takes an expression as a value is not required. You can improve performance by resolving such an expression on demand from within the connectors's code instead of resolving it automatically.
```